Wireframe/css/style.css

```diff
@@ -0,0 +1,71 @@
+body {
+    font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+    width: auto; 
+    margin-top: 40px;
+    background-color: beige;
+    margin-left: 120px;
+    margin-right: 100px;
+    margin-bottom: 50px;
+}
+
+.header {
+    align-items: center;
+    text-align: center;
+    padding-top: 2%;
+    padding-left: 3%;
+    padding-right: 3%;
+    padding-bottom: 3%;
+}
+
+.footer {
+    align-items: center;
+    text-align: center;
+    padding-top: 2%;
+    padding-left: 3%;
+    padding-right: 3%;
+    padding-bottom: 3%;
+}
+  
+}
+
+.article1 {
+    padding-top: 2%;
+    padding-left: 3%;
+    padding-right: 3%;
+    padding-bottom: 3%;
```

Comment:
You can simplify CSS like padding and margin into 1 line.

For example this is the same as what you have written above.
`padding: 2% 3% 3% 3%`
To break this down it would be `padding: top right bottom left`

It's not necessary, but in future can save you time and less code to write.